mibidata/mibi_image.py

```diff
@@ -110,7 +114,8 @@ def __init__(self, data, channels, run=None, date=None,
                  instrument=None, tissue=None, panel=None, version=None,
                  mass_offset=None, mass_gain=None, time_resolution=None,
                  miscalibrated=None, check_reg=None,
-                 datetime_format=_DATETIME_FORMAT, filename=None):
+                 datetime_format=_DATETIME_FORMAT, filename=None,
+                 optional_metadata=None):
```

Comment:
As we discussed offline, I think it would be better to continue adding attributes to the MibiImage class, such as description in this case. Then, instead of adding each as an optional argument to the constructor, implement kwargs and from the keywords match the values to attributes in the class.

Reply:
OK, I implemented `kwargs` for the metadata parameters in the constructor.